components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -239,7 +239,11 @@ class EditPostModal extends React.PureComponent {
     }
 
     handleKeyDown = (e) => {
-        if (this.props.ctrlSend && Utils.isKeyPressed(e, KeyCodes.ENTER) && e.ctrlKey === true) {
+        // listen for line break key combo and insert new line character
+        if (Utils.isUnhandledLineBreakKeyCombo(e)) {
+            e.stopPropagation(); // perhaps this should happen in all of these cases? or perhaps Modal should not be listening?
```

Comment:
this `stopPropogation` is necessary because `Modal` instance has duplicate listeners that otherwise result in this handler being called twice. It's probably worth evaluating whether `Modal` needs to be handling this key event redundantly and if there is a reason, it can likely be better implemented to avoid this unintuitive reuse of this event handler